### PR TITLE
Escape HTML entities in Youtube and Twitter handlers

### PR DESCRIPTION
**Pull request by sywhb**
- Escape youtube title and author name
- Escape HTML entities in Twitter title and description
- Escape HTML entities in puppeteer-parse

### CODE REVIEW

File: packages/content-fetch/youtube-handler.js [hunk 1]
```diff
@@ -36,11 +37,13 @@ exports.youtubeHandler = {
 
     const oembedUrl = `https://www.youtube.com/oembed?format=json&url=` + encodeURIComponent(`https://www.youtube.com/watch?v=${videoId}`)
     const oembed = (await axios.get(oembedUrl.toString())).data;
-    const title = oembed.title;
+    // escape html entities in title
+    const title = _.escape(oembed.title);
```
Comment (jacksonh) [inline packages/content-fetch/youtube-handler.js:41]: So if the input was `This "is quoted"` would the output here be `This \"is quoted\"`?

**Comment by sywhb** [comment]
Screenshot 2022-09-21 at 1 37 52 PM

File: packages/content-fetch/youtube-handler.js (hunk 1 above)
Comment (jacksonh) [reply packages/content-fetch/youtube-handler.js:41]: Oh and then we unescape when we set the title in the database?

**Comment by sywhb** [comment]
Screenshot 2022-09-23 at 4 50 13 PM

**Comment by jacksonh** [comment]
Awesome, thank you for verifying. Lots of encoding/decoding going on.

File: packages/content-fetch/youtube-handler.js (hunk 1 above)
Comment (sywhb) [reply packages/content-fetch/youtube-handler.js:41]: Yes, it will become `This \"is quoted\"` in the database

**Review by jacksonh — APPROVED** [review]

**Pull request merged**